timelock-impl/src/main/java/com/palantir/atlasdb/timelock/paxos/TimestampPaxosAcceptorResource.java

```diff
@@ -0,0 +1,99 @@
+/*
+ * (c) Copyright 2024 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.atlasdb.timelock.paxos;
+
+import com.palantir.atlasdb.metrics.Timed;
+import com.palantir.conjure.java.undertow.annotations.Handle;
+import com.palantir.conjure.java.undertow.annotations.HttpMethod;
+import com.palantir.paxos.BooleanPaxosResponse;
+import com.palantir.paxos.Client;
+import com.palantir.paxos.PaxosAcceptor;
+import com.palantir.paxos.PaxosPromise;
+import com.palantir.paxos.PaxosProposal;
+import com.palantir.paxos.PaxosProposalId;
+import javax.ws.rs.Consumes;
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+
+/**
+ * Encapsulates a {@link PaxosAcceptor} but at a defined path for routing purposes.
+ */
+@Path("/" + PaxosTimeLockConstants.INTERNAL_NAMESPACE
+        + "/" + PaxosTimeLockConstants.CLIENT_PAXOS_NAMESPACE
+        + "/{client: [a-zA-Z0-9_-]+}"
+        + "/acceptor")
+public final class TimestampPaxosAcceptorResource {
+    private final LocalPaxosComponents paxosComponents;
+
+    public TimestampPaxosAcceptorResource(LocalPaxosComponents paxosComponents) {
+        this.paxosComponents = paxosComponents;
+    }
+
+    @POST
+    @Path("prepare/{seq}")
```

Comment:
minor nit: before it seems like we were always starting `@Path` with a forward slash and we don't anymore `@Path("/foo") -> @Path("foo")`. I supposed we test this and both work fine? If so, any reason why we're prefer one convention over the other?

Reply:
I had stopped to think about this but was unsure what to go for here. The previous time we migrated we did not go for a consistent style (I think whatever reads these does the "correct" thing); see LockV1Resource and TimestampManagementResource in https://github.com/palantir/atlasdb/pull/6518/files